src/api/mod.rs

```diff
@@ -32,22 +32,3 @@ pub use context::propagation::{
     text_propagator::TextMapFormat, Extractor, Injector,
 };
 pub use context::Context;
-
```

Comment:
So `baggage` and `context` are feature flagged to trace but not under the `trace` module. Is there a strong opinion about whether these should be reorganized under the `trace` mod?

Reply:
Probably makes sense to keep the directory structure in line with https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md#api-package, but if features aren't used outside of trace then feature gating with `trace` probably makes sense. I think context is used by metrics though, so it should probably not be.

Reply:
Ok - I'll leave `baggage` and `context` alone.